docs/getting-started/licensing/license-and-legal.md

```diff
@@ -1,27 +1,44 @@
 ---
 category: licensing
 meta-title: Editor license | CKEditor 5 Documentation
-meta-description: Choosing the right licence type for you needs.
+meta-description: Choosing the right license type for you needs.
 menu-title: Editor license
 order: 30
 modified_at: 2024-11-20
 ---
 
 # Editor license and legal terms
 
-The following legal notices apply to CKEditor&nbsp;5 and all software from CKEditor&nbsp;5 Ecosystem included with it.
+CKEditor is dual-licensed. This means we make it available under a choice between an open source and a commercial license.
 
-Copyright (c) 2003–2024, CKSource Holding sp. z o.o. All rights reserved.
+## Open source license
 
-Licensed under the terms of [GNU General Public License Version 2 or later](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+The open source distribution of CKEditor is licensed under the terms of GPL 2+. If you plan to use CKEditor under this license, you can register your application using [the CKEditor GPL 2+ license registration form](https://info.ckeditor.com/l/1018062/2024-08-22/39jtg).
 
-## Free for Open Source
+If you are running an open source project with an OSS license incompatible with GPL please [contact us](https://ckeditor.com/contact/) – we will be happy to support your project with [a CKEditor 5 open source license that works for you](https://ckeditor.com/wysiwyg-editor-open-source/).
 
-If you are running an Open Source project under an OSS license incompatible with GPL, please [contact us](https://ckeditor.com/contact/). We will be happy to [support your project with a free CKEditor&nbsp;5 license](https://ckeditor.com/wysiwyg-editor-open-source/).
+**Note:** the CKEditor Free Plan is licensed under the GPL 2+ license.
 
-## Available commercial licenses
+## Commercial license
 
-TO DO
+The commercial license unlocks many benefits:
+
+* No GPL restrictions
+* Self-hosted CKEditor
+* Additionally paid Premium features such as:
+	* Asynchronous and Real-Time Collaboration
+	* Import from Word
+	* Export to Word
+	* Export to PDF
+	* Multi-level Lists
+	* Advanced Productivity
```

Comment:
Advanced productivity is not a name of our capability, where did it come from?

Reply:
![](https://github.com/user-attachments/assets/d8a86e24-f9f0-4d18-a79e-fd9742a42d63)

Reply:
![](https://github.com/user-attachments/assets/9f0399d1-746c-4f41-8d99-b203da7e35d2)

Ehh....